config/routes.rb

```diff
@@ -5,6 +5,7 @@
 
   resource :dashboard, only: [:index]
   resource :updates, only: [:create]
+  resources :nodes, only: [:destroy]
```

Comment:
I don't know. This is a matter for the future I'd say. The dashboard and the setup controllers do not really follow any best practices, and we simply dumped methods into those. After the first release we'll have to re-organize things a bit.

For this PR, having this functionality here makes sense (and the "adding nodes" feature could re-use it but then again, it's a matter for re-organization in the future 😉).